src/oaklib/converters/data_model_converter.py

```diff
@@ -10,10 +10,16 @@
 
 @dataclass(eq=False)
 class DataModelConverter(ABC):
-    """Base class for all inter-data model converters."""
+    """Base class for all inter-data model converters.
+
+    Do not use this directly: use one of the subclasses.
+    """
 
     curie_converter: curies.Converter = None
+    """Converts between CURIEs and URIs"""
+
     labeler: Callable[[CURIE], Optional[str]] = None
+    """A function that returns a label for a given CURIE"""
```

Comment:
Codestyle: I guess this is a docstring and not a normal comment `#` because it is a `Callable`